### PR TITLE
DOCS-1407: Add viam module generator / README template to create page

### PR DESCRIPTION
**Pull request by sguequierre**
* Add instruction to use generator (necessary to fully use template) and instruction that just points to template

### CODE REVIEW

File: docs/registry/create/_index.md [hunk 1]
```diff
@@ -1276,13 +1280,95 @@ For more information on building a module in C++, see the [C++ SDK Build Documen
 {{% /tab %}}
 {{% /tabs %}}
 
+## (Optional) create a README
+
+To provide usage instructions for any modular resources in your module, you should create a <file>README.md</file> file following this template:
+
+{{% expand "Click to view template" %}}
+
+Strings surrounded by `<%= %>` indicate variables that you should replace with the appropriate values.
```
Comment (npentrel) [inline docs/registry/create/_index.md:1289]: I think it makes sense to indicate variables that need replacing in the same way we do across code samples in the docs.

File: docs/registry/create/_index.md [hunk 2]
```diff
@@ -1276,13 +1280,95 @@ For more information on building a module in C++, see the [C++ SDK Build Documen
...
+# <%= name %> modular service
+
+This module implements the [<%= api_namespace %> <%= api_name_lower %> API](https://github.com/<%= api_namespace %>/<%= api_name_lower %>-api) in a <%= model %> model.
+With this model, you can...
+
+## Requirements
+
+_Add instructions here for any requirements._
+
+```bash
```
Comment (npentrel) [inline docs/registry/create/_index.md:1302]: I'm not sure if you need to escape these?

File: docs/registry/create/_index.md [hunk 3]
```diff
@@ -1276,13 +1280,95 @@ For more information on building a module in C++, see the [C++ SDK Build Documen
...
+
+```
+````
```
Comment (npentrel) [inline docs/registry/create/_index.md:1305]: what are these 4 here?

**Review by npentrel — CHANGES_REQUESTED** [review]
please make sure PRs build before sending for review

File: docs/registry/create/_index.md [hunk 4]
```diff
@@ -1276,13 +1280,95 @@ For more information on building a module in C++, see the [C++ SDK Build Documen
...
+
+To provide usage instructions for any modular resources in your module, you should create a <file>README.md</file> file following this template:
+
+{{% expand "Click to view template" %}}
+
+Strings surrounded by `<%= %>` indicate variables that you should replace with the appropriate values.
+If you follow [the instructions to generate module scaffolding for a Python module](#optional-generate-your-python-module-scaffolding), the template file is generated for you with the environment variables filled in.
+
+````md
+# <INSERT NAME> modular service
```
Comment (sguequierre) [inline docs/registry/create/_index.md:1293]:
```suggestion
# <INSERT NAME> modular resource
```

File: docs/registry/create/_index.md [hunk 5]
```diff
@@ -1051,6 +1051,90 @@ We recommend that you use a C++ logging library to assist with log message forma
 {{% /tab %}}
 {{< /tabs >}}
 
+## (Optional) create a README
+
+To provide usage instructions for any modular resources in your module, you should create a <file>README.md</file> file following this template:
+
+{{% expand "Click to view template" %}}
+
+Strings surrounded by `<%= %>` indicate variables that you should replace with the appropriate values.
```
Comment (npentrel) [inline docs/registry/create/_index.md:1060]:
```suggestion
Strings of the form `<INSERT X>` indicate variables that you should replace with the appropriate values.
```

**Review by npentrel — APPROVED** [review]
LGTM % remaining change

**Pull request merged**